auth-web/src/views/auth/BusinessProfileView.vue

```diff
@@ -107,7 +107,7 @@ export default class BusinessProfileView extends Mixins(AccountChangeMixin, Next
         this.$router.push('/home')
       }
     } else {
-      const businessUrl = `${ConfigHelper.getBusinessURL()}${this.currentBusiness.businessIdentifier}`
+      const businessUrl = `${ConfigHelper.getBusinessDashURL()}${this.currentBusiness.businessIdentifier}`
```

Comment:
To test this:

1. login as a regular user
2. go to any active business
3. go to Edit Profile
4. click Back icon button